mailer.js

```diff
@@ -129,12 +132,12 @@ var sendMail = function (req, res, next) {
             console.error(err);
             res.json({error: err.message});
         } else {
-            if (json.message === 'success') {
+            if (json[0].statusMessage === 'Accepted') {
```

Comment:
why only check the first index?

Reply:
as there is only one entry as we only send one email at a time :)